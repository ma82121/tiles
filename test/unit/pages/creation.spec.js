import { shallowMount, createLocalVue } from '@vue/test-utils'
import Vuex from 'vuex'
import firebase from '@/plugins/firebase'
import Creation from '@/pages/creation'
import {
  state as configState,
  mutations as configMutations,
  getters as configGetters,
  actions as configActions
} from '@/store/config'
import {
  state as menuState,
  mutations as menuMutations,
  getters as menuGetters,
  actions as menuActions
} from '@/store/menu'
import {
  state as authState,
  mutations as authMutations,
  getters as authGetters,
  actions as authActions
} from '@/store/auth'

const localVue = createLocalVue()
localVue.use(Vuex)

describe('Creation', () => {
  let store
  beforeEach(() => {
    store = new Vuex.Store({
      modules: {
        config: {
          namespaced: true,
          state: configState,
          mutations: configMutations,
          getters: configGetters,
          actions: configActions
        },
        menu: {
          namespaced: true,
          state: menuState,
          mutations: menuMutations,
          getters: menuGetters,
          actions: menuActions
        },
        auth: {
          namespaced: true,
          state: authState,
          mutations: authMutations,
          getters: authGetters,
          actions: authActions
        }
      }
    })
  })

  test('snapshot', () => {
    const wrapper = shallowMount(Creation, {
      store,
      localVue
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
