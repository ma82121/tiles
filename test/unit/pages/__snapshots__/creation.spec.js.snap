// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Creation snapshot 1`] = `
<div
  class="creation-page"
>
  <bg-stub
    isfixed="true"
    type="1"
  />
   
  <menu-stub />
   
  <div
    class="header-title"
  >
    Creation
  </div>
   
  <!---->
</div>
`;
